### ISSUE TITLE
Errors raised during installation

### ISSUE THREAD

**Issue by ahlusar1989** [coreysharris/TotalImage#1]
I am attempting to follow the instructions for installation:

Here is the version I am using installed on **Mac OSX High Sierra version 10.13.1**:

**Macaulay2-1.13-x86_64-MacOS-10.14.2.**

When I execute the following I receive this:

```
i1 :  installPackage "TotalImage"
--warning: symbol "Tries" in ReesAlgebra.Dictionary is shadowed by a symbol in TotalImage.Dictionary
--  use the synonym ReesAlgebra$Tries
stdio:1:2:(3): error: mutable unexported unset symbol(s) in package TotalImage: 'eliminate'
TotalImage.m2:196:19-196:28: here is the first use of 'eliminate'
```

Any thoughts on what I am missing here?

**Comment by coreysharris** [comment]
Ah, two breaking changes were introduced in Macaulay2 1.13.  This one is because we now need to declare that we are using the automatically-loaded Elimination package. I've fixed this in bdaca29c.

**Comment by ahlusar1989** [comment]
Thank you @coreysharris!
